client/src/views/home/SummaryStep.vue

```diff
@@ -0,0 +1,202 @@
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+
+    <!-- fullname -->
+    <ion-item
+      class="summary-item"
+    >
+      <ion-label class="summary-item__label subtitles-sm">
+        {{ $t('CreateOrganization.overview.fullname') }}
+      </ion-label>
+      <ion-text class="summary-item__text body">
+        {{ fullname }}
+      </ion-text>
+      <ion-button
+        fill="clear"
+        class="summary-item__button"
+        @click="$emit('update-request', OrgInfo.Fullname)"
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+
+    <!-- Email -->
+    <ion-item
+      class="summary-item"
+    >
+      <ion-label class="summary-item__label subtitles-sm">
+        {{ $t('CreateOrganization.overview.email') }}
+      </ion-label>
+      <ion-text class="summary-item__text body">
+        {{ email }}
+      </ion-text>
+      <ion-button
+        fill="clear"
+        class="summary-item__button"
+        @click="$emit('update-request', OrgInfo.Email)"
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+
+    <!-- deviceName -->
+    <ion-item
+      class="summary-item"
+    >
+      <ion-label class="summary-item__label subtitles-sm">
+        {{ $t('CreateOrganization.overview.deviceName') }}
+      </ion-label>
+      <ion-text class="summary-item__text body">
+        {{ deviceName }}
+      </ion-text>
+      <ion-button
+        fill="clear"
+        class="summary-item__button"
+        @click="$emit('update-request', OrgInfo.DeviceName)"
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+
+    <!-- serverMode -->
+    <ion-item
+      class="summary-item"
+    >
+      <ion-label class="summary-item__label subtitles-sm">
+        {{ $t('CreateOrganization.overview.server') }}
+      </ion-label>
+      <ion-text class="summary-item__text body">
+        {{ serverMode }}
+      </ion-text>
+      <ion-button
+        fill="clear"
+        class="summary-item__button"
+        @click="$emit('update-request', OrgInfo.ServerMode)"
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+
+    <!-- fullname -->
+    <ion-item
+      class="summary-item"
+    >
+      <ion-label class="summary-item__label subtitles-sm">
+        {{ $t('CreateOrganization.overview.password') }}
+      </ion-label>
+      <ion-text class="summary-item__text body">
+        {{ password }}
+      </ion-text>
+      <ion-button
+        fill="clear"
+        class="summary-item__button"
+        @click="$emit('update-request', OrgInfo.Password)"
+      >
+        {{ $t('CreateOrganization.button.modify') }}
+      </ion-button>
+    </ion-item>
+  </ion-list>
+</template>
+
+<script lang="ts">
+export enum OrgInfo {
+  Organization = 'organization',
+  Fullname = 'fullname',
```

Comment:
Since the values targetting the user are all used in the same place anyway, the enum could be shorten into something like

```
enum NewOrgInfo {
  Organization,
  UserInfo, // includes email, name, device name
  Password,
  Server,
}
```